src/views/tutorials-landing/index.tsx

```diff
@@ -192,53 +195,8 @@ const renderProductSections = (productSlugs, pageContent) => {
 	})
 }
 
-const BetterTogetherSection = () => {
-	return (
-		<section className={s.betterTogetherSection}>
-			<h2
-				style={{
-					fontSize: '1.875rem',
-					fontWeight: 700,
-					lineHeight: '1.3',
-					maxWidth: '40%',
-					marginBottom: 36,
-				}}
-			>
-				{BETTER_TOGETHER_SECTION_TITLE}
-			</h2>
-			<ul
-				style={{
-					margin: 0,
-					padding: 0,
-					display: 'flex',
-					gap: 22,
-					alignItems: 'flex-start',
-					listStyle: 'none',
-				}}
-			>
-				{BETTER_TOGETHER_SECTION_COLLECTION_SLUGS.map((collectionSlug) => {
-					return (
-						<li
-							key={collectionSlug}
-							style={{
-								backgroundColor: '#DBDBDC',
-								border: '1px solid magenta',
-								width: 400,
-								height: 300,
-								borderRadius: 10,
-								padding: 24,
-							}}
-						>
-							{collectionSlug}
-						</li>
-					)
-				})}
-			</ul>
-		</section>
-	)
-}
-
 const TutorialsLandingView = ({ pageContent }: $TSFixMe) => {
```

Comment:
I think we should type this pageContent sooner rather than later.  I would benefit from the explicitness.

Reply:
It's coming! 😊